lib/modules/keyboardNav.js

```diff
@@ -1104,6 +1113,12 @@ function reply(selected = getSelected()) {
 	));
 }
 
+function edit(selected = getSelected()) {
+	if (selected.isComment() || selected.isSelfPost()) {
```

Comment:
Probably no need to have this conditional, as `.edit-usertext` will be available only things which can be edited.